### PR TITLE
Session and Authentication projet

### PR DESCRIPTION
**Pull request by crappypapa**
This highlights
- Using Devise gem to enable authentication in rails.
- Unless Logged in, visitors cannot edit, or create posts or see the author of posts
- Logged in Users can only edit or destroy their own posts

### CODE REVIEW

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
### REQUESTED CHANGES 🛑
**Hey @crappypapa and @CollinsTatang, you have really done a good job so far. Let's just fix few issues so that we can move**
- [ ] As a logged-in user, I should log out, but When I try to log out I get the error in the image below. Please fix it.

signout

- [ ] Check other comments and fix them.
[OPTIONAL] you can style your project so that it looks good.

When you are done, submit another review request.✔💪
In case you have a question, please reach me on slack as Elisha kyakopo

**Review by henatan99 — CHANGES_REQUESTED** [review]
# Changes Required :recycle: 
Hi there, you did an amazing job so far :+1: :clap: :clap: . There is only one change you need to address which was already mentioned in the previous review. 

## To Highlight 
- You have implemented a working RESTfull web app and hats off for that. 
## To Improve 
- Even though, you have implemented a good RESTful web app, you have used all the views and resources, which are redundant. You are supposed to use [:new, :create, : index] routes methods and corresponding view files. It doesn't mean that you are wrong, but it is the requirement of the project as it helps you develop best practices skills. Please see the attached screenshot below:
[ScreenShot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/authentication/posts_routes_requirement.png) 

## OPTIONAL Improvements 
- Please see a couple of inline comments which point to some semantic (naming) aspects. This is an optional comment you can add to your project. 
- In the Posts controllers the before_action method should also require sign_in for the posts [:new, :create] actions only. You can make these changes to make your project more professional but it is not a mandatory requirement for submission of your project. 

### Kindly make the requested changes and submit for another review.

Should you have any enquiry, please reach out to me via:
[LinkedIn](https://www.linkedin.com/in/henok-mekonnen/) | [Twitter](https://twitter.com/henatan99) | Henok Mekonnen on Slack 

## Happy Coding :computer:

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -8,9 +8,82 @@
...
-  </head>
+  <%= stylesheet_pack_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+</head>
 
   <body>
+
+  <!--<header>
+      <nav class="navbar navbar-expand-md navbar-dark bg-dark fixed-top ">
+      <div class="navbar-brand ">
+        <%= link_to 'MEMBERS', root_path %>
```
[the remark this answers is not in the document]
Comment (crappypapa) [reply app/views/layouts/application.html.erb:19]: Ideally, it should link to the landing page of our application (root_path) but as that is not being implemented, we decided to make do with this option.

**Comment by crappypapa** [comment]
image


I am sorry about the accidental deletion of the comment. It was a suggestion by @henatan99 to make changes to the line highlighted in the image above. Changes which I have implemented.

**Pull request merged**